examples/custom-config/main.tf

```
@@ -3,7 +3,7 @@ provider "lacework" {}
 provider "aws" {}
 
 module "aws_config" {
-  source                    = "../../"
+  source                    = "lacework/config/aws"
```

Unfortunately, since these are "living tests", we should continue to point them to `../..` instead. 
```suggestion
  source                    = "../.."
```